drivers/slipmux/include/slipmux_internal.h

```diff
@@ -0,0 +1,202 @@
+/*
+ * SPDX-FileCopyrightText: 2018 Freie Universität Berlin
+ * SPDX-FileCopyrightText: 2025 HAW Hamburg
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+#pragma once
+
+/**
+ * @ingroup drivers_slipmux
+ * @{
+ *
+ * @file
+ * @internal
+ * @brief   Internal SLIPMUX device definitions
+ *
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ * @author  Bennet Hattesen
+ */
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "periph/uart.h"
+#include "mutex.h"
+
+#include "slipmux_params.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Number of slipmux devices, Parameters and array defined in slipmux_params.h
+ */
+#define SLIPMUX_DEV_NUM ARRAY_SIZE(slipmux_params)
+
+/**
+ * @brief   List of slipmux devices.
+ */
+extern slipmux_t slipmux_devs[SLIPMUX_DEV_NUM];
+
+/**
+ * @name    SLIP marker bytes
+ * @see     [RFC 1055](https://tools.ietf.org/html/rfc1055)
+ * @{
+ */
+/**
+ * @brief   Ends a frame
+ */
+#define SLIPMUX_END         (0xc0U)
+/**
+ * @brief   Initiates escape sequence
+ */
+#define SLIPMUX_ESC         (0xdbU)
+/**
+ * @brief   Ends escape sequence, next byte is END
+ */
+#define SLIPMUX_END_ESC     (0xdcU)
+/**
+ * @brief   Ends escape sequence, next byte is ESC
+ */
+#define SLIPMUX_ESC_ESC     (0xddU)
+/** @} */
+
+/**
+ * @name    SLIPMUX marker bytes
+ * @see     [SLIPMUX](https://tools.ietf.org/html/draft-bormann-t2trg-slipmux-03#section-4)
+ * @{
+ */
+/**
+ * @brief   Starts a diagnostic frame
+ */
+#define SLIPMUX_STDIO_START (0x0aU)
```

Comment:
```suggestion
#define SLIPMUX_START_STDIO (0x0aU)
```

and similar below would sort them more nicely, e.g. in the documentation.